osprey_flask_app/routes.py

```diff
@@ -1,22 +1,24 @@
 """Defines all routes available to Flask app"""
 
-from flask import Blueprint, request, send_file
+from flask import Blueprint, request, Response, url_for
 from .run_rvic import run_full_rvic
-from .utils import create_full_arg_dict
-from pywps.app.exceptions import ProcessError
+from .utils import create_full_arg_dict, inputs_are_valid
 
-import os
 import requests
-from tempfile import NamedTemporaryFile
+import concurrent.futures
+import uuid
 
-data = Blueprint("data", __name__, url_prefix="/data")
+osprey = Blueprint("osprey", __name__, url_prefix="/osprey")
+pool = concurrent.futures.ThreadPoolExecutor(max_workers=1)
```

Comment:
Maybe `max_workers` should be configurable by an environment variable? I can imagine we'd want to set that differently in different execution environments.

Reply:
Is it something that would be set in the `Makefile`?

Reply:
No, use something like `os.environ.get("MAX_WORKERS", 1)` (That retrieves the value of the environment variable MAX_WORKERS from the operating system, but uses 1 as a default, if it's unset) and then document in the README that whoever deploys the service should set this variable.